src/common/types/profile.js

```diff
@@ -115,3 +115,13 @@ export type Profile = {
   tasktracer: TaskTracer,
   threads: Thread[],
 };
+
+export type Node = {
```

Comment:
This file should only contain the canonical preprocessed profile types. There is a separate profile-derived.js file for types that are derived off of the main format. I should probably add some docs here explaining that.